activerecord/lib/active_record/railtie.rb

```diff
@@ -142,10 +142,11 @@ class Railtie < Rails::Railtie # :nodoc:
         config.after_initialize do |app|
           ActiveSupport.on_load(:active_record) do
             db_config = ActiveRecord::Base.configurations.configs_for(env_name: Rails.env).first
+            next if db_config.nil?
```

Comment:
Let's not handle `nil`, especially since we're always taking the first config. There's nothing to go to next anyway. So in this PR just change the line below to remove the safe navigation.

Reply:
Done